service/api.go

```diff
@@ -273,6 +273,13 @@ func (as *APIServer) AttachTestLog(w http.ResponseWriter, r *http.Request) {
 	log.Task = t.Id
 	log.TaskExecution = t.Execution
 
+	grip.Debug(message.Fields{
+		"message":    "received test log",
+		"task":       t.Id,
```

Comment:
I would maybe add the task's project and requester as well to help aggregate stats in both places (and other fields if you might group by them)

Reply:
done